src/app/accounts/set-default-walletid.ts

```diff
@@ -0,0 +1,22 @@
+import { InvalidWalletId } from "@domain/errors"
+import { AccountsRepository } from "@services/mongoose"
+
+export const setDefaultWalletId = async ({
```

Comment:
Where does this get called? Can it work with a persisted wallet?

Reply:
only in tests for now